themes/nav-community-v1/layouts/shortcodes/box_links_section_two.html

```diff
@@ -9,7 +9,7 @@ <h2 class="themeix-title-h1" style="color: {{ .Get "titleFontColor" }}">{{ .Get
                     </div>
                 </div>
                 <div class="row">
-                    <div class="col-md-4" style="width: 50%;">
```

Comment:
I'm not sure who coded this line, but it's definitely not the right way to do this. 

At the least this should be col-md-6 to become 50% width. Telling it to be 1/3 width with col-md-4 and then overriding this to 50% with inline css lazy coding.

The other thing is that 50% width is way too wide and inconsistent with how 2 columns have been displayed elsewhere on this site and on NavHub. It should be using col-md-4 with an offset of 2 so it becomes center aligned as it is now.

Please get familiar with the bootstrap CSS 3.3.7 grid system: https://getbootstrap.com/docs/3.3/css/#grid